### ISSUE TITLE
Deleting item from vomnibar history

### ISSUE THREAD

**Issue by avmohan** [philc/vimium#2550]
Is there any way to remove a past search query from vomnibar suggestions?

**Comment by Beasta** [comment]
yeah i've got this issue where i've accidentally pasted some giant strings into the omnibar. whenever i search something thats in one of those it slows to a crawl. i would like to delete those large strings from history.

**Comment by philc** [comment]
I also very much want this feature. I'd gladly accept a PR.

**Comment by philc** [comment]
For inspiration: Chrome's Omnibox has this capability. `Shift-Delete` will delete the selected item from history. Also, if you hover over an item with the mouse, an "X" button appears; clicking it deletes the item from history.

If the selection is a tab, it should be closed.
